samples/subsys/mgmt/mcumgr/smp_svr/sample.yaml

```diff
@@ -43,6 +43,12 @@ tests:
       - mg100
     integration_platforms:
       - nrf52840dk/nrf52840
+  sample.mcumgr.smp_svr.serial_shell_bt:
+    extra_args: OVERLAY_CONFIG="overlay-shell.conf;overlay-bt.conf"
+    platform_allow:
+      - nrf54l15pdk_nrf54l15
+    integration_platforms:
+      - nrf54l15pdk_nrf54l15
```

Comment:
Add to the `sample.mcumgr.smp_svr.bt` test instead of making a new one, overlay-bt.conf has the configuration from overlay-shell.conf